nbqa/__main__.py

```diff
@@ -358,7 +358,9 @@ def _run_command(
     if "No module named" in err:
         raise ValueError(
             f"Command `{command}` not found. "
-            "Please make sure you have it installed before running nbqa on it."
+            "Please make sure you have it installed in the same environment as nbqa.\n"
+            "See https://realpython.com/python-virtual-environments-a-primer/ for how to "
```

Comment:
I would change it to `I.e. see`, since there are lots of ways to setup a venv i.e. `conda` is missing in that guide.